### DISCUSSION TITLE
Settings for crossfade_time, image_order, and image_animation_ken_burns

### DISCUSSION THREAD

**Discussion by lennon101** [damongolding/immich-kiosk#88] [Feature request]
**Is your feature request related to a problem? Please describe.**
Nope. No problem here! 

**Describe the solution you'd like**
In much the same way that you can apple these settings in the lovelace-wallpanel [config](https://github.com/j-a-n/lovelace-wallpanel?tab=readme-ov-file#configuration), I'd love to see these settings in immich-kiosk:

- crossfade_time: Crossfade duration in seconds for screensaver images.
- image_order: The order in which the images are displayed (possible values are: sorted / random).
- image_animation_ken_burns: Apply a Ken Burns effect (panning and zooming) to the images
- image_animation_ken_burns_zoom: Zoom level for the Ken Burns effect.

**Describe alternatives you've considered**
Not sure if there are settings already in the pipeline for these features or perhaps they have already been implemented and I missed them in the documentation? 

**Additional context**
No furture info needed at this stage.

**Comment by damongolding** [comment]
* I think I can implement the cross-fade duration relatively easily 
* At the moment it's not really possible to implement a sorted option (easily) as Kiosk was built with serving random images. It's possible but would require a fair bit of logic reworking.
* Adding a Kens Burn effect has been requested a fair amount so I will likely add it at some point 🙂

**Comment by damongolding** [comment]
Implemented crossfade and fade transition duration options in 0.9.0

**Comment by kelvinburke** [comment]
+1 for non-random ordering!
